src/@commonsku/styles/LightIndicator.tsx

```diff
@@ -55,7 +77,10 @@ const LightIndicatorText = styled.p<LightIndicatorTextType>`
         font-size: ${props => props.large ? themeOptions.fontStyles.p.medium.fontSize : themeOptions.fontStyles.p.small.fontSize};
         font-family: ${themeOptions.fontStyles.p.small.fontFamily};
         line-height: ${themeOptions.fontStyles.p.small.lineHeight};
-        color: ${props => props.LightIndicatorTextColor ? colors.neutrals.bodyText : colors.neutrals[70]};
+        color: ${props => {
+        if (props.LightIndicatorTextColor === true) return colors.neutrals.bodyText;
+        else return colors.neutrals[70]
```

Comment:
```suggestion
        return props.LightIndicatorTextColor ? colors.neutrals.bodyText : colors.neutrals[70];
```
or braces would work too
```suggestion
        if (props.LightIndicatorTextColor === true) {
            return colors.neutrals.bodyText;
        }
        return colors.neutrals[70];
```